packages/kuma-gui/src/app/services/views/MeshServiceDetailView.vue

```diff
@@ -123,13 +123,109 @@
+              mesh: route.params.mesh,
+              serviceType: 'meshservices',
+              serviceName: route.params.service,
+            })"
+          >
+            <template #loadable="{ data: hostnames }">
+              <DataCollection
+                type="hostnames"
+                :items="hostnames?.items ?? [undefined]"
+              >
+                <AppCollection
+                  type="hostnames-collection"
+                  data-testid="hostnames-collection"
+                  :items="hostnames?.items"
+                  :headers="[
+                    { ...me.get('headers.hostname'), label: t('services.detail.hostnames.hostname'), key: 'hostname' },
+                    { ...me.get('headers.zones'), label: t('services.detail.hostnames.zone'), key: 'zones' },
+                  ]"
+                  @resize="me.set"
+                >
+                  <template #hostname="{ row: item }">
+                    <b>
+                      <XCopyButton
+                        :text="item.hostname"
+                      />
+                    </b>
+                  </template>
+                  <template #zones="{ row: item }">
+                    <XLayout type="separated">
+                      <XBadge
+                        v-for="(zone, index) of item.zones"
+                        :key="index"
+                        appearance="decorative"
+                      >
+                        <XAction
+                          :to="{
+                            name: 'zone-cp-detail-view',
+                            params: {
+                              zone: zone.name,
+                            },
+                          }"
+                        >
+                          {{ zone.name }}
+                        </XAction>
+                      </XBadge>
+                    </XLayout>
+                  </template>
+                </AppCollection>
+              </DataCollection>
+            </template>
+          </DataLoader>
+        </XCard>
+
+        <XCard>
+          <template #title>
+            {{ t('services.detail.dpp-status.title') }}
+          </template>
+
+          <XLayout
+            type="columns"
+            class="columns-with-borders"
+          >
+            <ResourceStatus
+              :total="props.data.status.dataplaneProxies.total"
+              :online="props.data.status.dataplaneProxies.connected"
+              data-testid="connected-dpps"
+            >
+              <template #icon>
+                <XIcon name="connected" />
+              </template>
+              <template #title>
+                {{ t('services.detail.dpp-status.connected') }}
+              </template>
+            </ResourceStatus>
+
+            <ResourceStatus
+              :total="props.data.status.dataplaneProxies.healthy"
+              data-testid="healthy-dpps"
+            >
+              <template #icon>
+                <XIcon name="heartbeat" />
```

Comment:
nit: should the name of the icon be `health` or `healthy` to describe what we use it for rather than what it is?